### PR TITLE
Add word of the day section

### PR DESCRIPTION
**Pull request by alireza-jahandoost**
### What does this PR do?
Add "Word of the day" section to index page

### Any new dependencies? Why were they added?
No

### Does is close any issue(s)?
[issue #157](https://github.com/narbs91/glosseta/issues/157)

Closes # ...

### CODE REVIEW

**Comment by narbs91** [comment]
Hi @alireza-jahandoost thank you so much for the prompt work on this issue!  It looks like you are getting some build issues with the page on build you can observe the output [here](https://github.com/narbs91/glosseta/runs/6001325348?check_suite_focus=true) or look to reproduce it via `yarn build` on your machine.

**Comment by alireza-jahandoost** [comment]
@narbs91 The problem solved with moving new component to src/components directory. I think we have to move other components from src/pages directory to this directory. Should I open an issue about it?

**Comment by narbs91** [comment]
Hi @alireza-jahandoost moving it outside the pages folder seems kinda odd for it to be the fix.  I'll have a closer look in the upcoming days as well as a review of the UI (from my original design proposal) and let you know.  Thank you for the great work 🚀

**Comment by alireza-jahandoost** [comment]
Hi @narbs91 . If you reset the HEAD to the point that it has error, and try to build, a link to the cause of error is [this](https://nextjs.org/docs/messages/prerender-error). And the recommended way to solve it is to move every non-pages to out of `page` directory.

**Review by narbs91 — CHANGES_REQUESTED** [review]
Overall looks great!  Just a few minor tweaks and we should be good to go

File: src/components/daily-word/daily-word.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (alireza-jahandoost) [reply src/components/daily-word/daily-word.tsx:1]: Yes, of course.

**Review by narbs91 — APPROVED** [review]
LGTM!  Awesome work here and really appreciate you stepping up to do this, i think people are really going like this feature :).  I'll get this merged in the morning tomorrow!

**Comment by narbs91** [comment]
@all-contributors please add @alireza-jahandoost for code

**Pull request merged**